lib/recurly/utils.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace Recurly;
+
+class Utils
+{
+    private static $_client_version = \Recurly\Version::CURRENT;
+
+    public static function getUserAgent(): string
```

Comment:
It doesn't really matter, but these 2 functions could be moved to `RecurlyTraits` (which should probably be renamed). Traits being PHP's version of mixins

Reply:
I like that better, missed that pattern.